.travis.yml

```diff
@@ -15,7 +15,8 @@ env:
 
 before_install:
   - git fetch --tags
-  - docker pull exasol/docker-db:7.0.4
+  - docker pull exasol/docker-db:7.0.5
```

Comment:
Should we use the latest 7.0.6?

Reply:
Good idea!
Forgot that we had another release.